app/src/main/java/net/bible/android/view/activity/bookmark/Bookmarks.kt

```diff
@@ -266,9 +267,9 @@ class Bookmarks : ListActivityBase(), ActionModeActivity {
 
     private fun changeBookmarkSortOrder() {
         bookmarkSortOrder = when (bookmarkSortOrder) {
-            BookmarkSortOrder.BIBLE_ORDER -> BookmarkSortOrder.CREATED_AT
-            BookmarkSortOrder.CREATED_AT -> BookmarkSortOrder.BIBLE_ORDER
-            else -> BookmarkSortOrder.CREATED_AT
+            BookmarkSortOrder.BIBLE_ORDER -> BookmarkSortOrder.CREATED_AT_DESC
+            BookmarkSortOrder.CREATED_AT_DESC -> BookmarkSortOrder.BIBLE_ORDER
+            else -> BookmarkSortOrder.CREATED_AT_DESC
```

Comment:
Shouldn't this be full loop between BIBLE_ORDER -> CREATED_AT_DESC -> CREATED_AT -> BIBLE_ORDER? Now we don't go ever to CRATED_AT.

Reply:
```suggestion
            BookmarkSortOrder.BIBLE_ORDER -> BookmarkSortOrder.CREATED_AT_DESC
            BookmarkSortOrder.CREATED_AT_DESC -> BookmarkSortOrder.CREATED_AT            
            BookmarkSortOrder.CREATED_AT -> BookmarkSortOrder.BIBLE_ORDER
            else -> BookmarkSortOrder.CREATED_AT_DESC
```